packages/sources/coincodex/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'CoinCodex'
+
+export const DEFAULT_BASE_URL = 'https://coincodex.com/api/coincodex/get_coin/'
```

Comment:
This base URL looks like it tied to a specific API endpoint of `get_coin`. We should keep this as base level as possible and handle that in the endpoint code. (Just `adapter.ts` in this case).

Reply:
So then DEFAULT_BASE_URL should just be "https://coincodex.com/api/coincodex/" and then append that get_coin endpoint in adapter.ts?

Reply:
Exactly. Though in the case of the endpoint it would be prepended, so like 
```
 const options = {
...config.api,
baseUrl: config.api.baseURL,
url: `get_coin/${base}`
}
```